src/test/java/com/acrolinx/sidebar/utils/XMLLookupUtilsTest.java

```diff
@@ -5,51 +5,98 @@
 import static org.junit.jupiter.api.Assertions.assertTrue;
 
 import com.acrolinx.sidebar.pojo.document.IntRange;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
 import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
 import org.junit.jupiter.api.Test;
 
 class XMLLookupUtilsTest {
-  private static final String XML_CONTENT =
-      "<?xml version=\"1.0\" encoding=\"utf-8\"?>\n"
-          + "<bookstore xmlns:xsi=\"http://www.w3.org/2001/XMLSchema-instance\"\n"
-          + "           xsi:noNamespaceSchemaLocation=\"BookStore.xsd\">\n"
-          + "    <book price=\"730.54\" ISBN=\"string\" publicationdate=\"2016-02-27\">\n"
-          + "        <title>Some hideous story about monsters</title>\n"
-          + "        <author>\n"
-          + "            <first-name>Max</first-name>\n"
-          + "            <last-name>Muster</last-name>\n"
-          + "        </author>\n"
-          + "        <genre>Fantastic Fantasy</genre>\n"
-          + "    </book>\n"
-          + "    <book price=\"6738.774\" ISBN=\"string\">\n"
-          + "        <title>Little munchkins on adventures</title>\n"
-          + "        <author>\n"
-          + "            <first-name>Susi</first-name>\n"
-          + "            <last-name>Some Name</last-name>\n"
-          + "        </author>\n"
-          + "    </book>\n"
-          + "</bookstore>\n";
-
-  @Test
-  void findOffsetInXmlStringByXpath() {
+  private static String readFileContent(String path) throws IOException {
+    try (Stream<String> lines = Files.lines(Paths.get(path))) {
+      return lines.collect(Collectors.joining("\n"));
+    }
+  }
+
+  private static String getXmlContent() throws IOException {
+    return readFileContent("src/test/resources/xml/xml-with-no-namespace.xml");
+  }
+
+  @Test
+  void findOffsetInXmlStringByXpathWithNamespaceAndEmptyTagAndComment() throws IOException {
+    final String xmlPath = "//proc:procedure[1]/p[4]";
```

Comment:
I think most people just call it "XPath", so I would prefer the name "`xPath`" (or even better `xPathString`, since `javax.xml.xpath.XPath` is a different type)